phpstan.neon

```diff
@@ -1,18 +1,15 @@
 includes:
     - vendor/phpstan/phpstan-nette/extension.neon
     - vendor/phpstan/phpstan-nette/rules.neon
+    - vendor/eloquent/phpstan-phony/etc/phpunit.neon
 
 parameters:
     excludes_analyse:
         - %rootDir%/../../../tests/_*
 
     ignoreErrors:
-        - '#^Access to an undefined property Eloquent\\Phony\\Mock\\Handle\\InstanceHandle::\$[a-zA-Z]++#'
-        - '#^Parameter \#[0-9]++ \$[a-zA-Z]++ of class [a-zA-Z\\]++ constructor expects [a-zA-Z\\]++, Eloquent\\Phony\\Mock\\Mock given#'
-        - '#^Parameter \#[0-9]++ \$[a-zA-Z]++ of method [a-zA-Z\\:]++\(\) expects [a-zA-Z\\|]++, Eloquent\\Phony\\Mock\\Mock given#'
-        - '#^Method [a-zA-Z\\]++::[a-zA-Z]++\(\) should return [a-zA-Z\\]++ but returns Eloquent\\Phony\\Mock\\Mock#'
+        - '#^Access to an undefined property Eloquent\\Phony\\Mock\\Handle\\InstanceHandle(<[^>]+>)?::\$[a-zA-Z]++#'
```

Comment:
This is necessary until I can "fix" instance handle magic properties. The error message can now look like:

```
 ------ -------------------------------------------------------------------------
  Line   tests/unit/src/EitherRuleHandlerTest.php
 ------ -------------------------------------------------------------------------
  76     Access to an undefined property
         Eloquent\Phony\Mock\Handle\InstanceHandle<Arachne\Verifier\RuleInterfac
         e>::$getCode.
 ------ -------------------------------------------------------------------------
```

with generics-style notation of the type.

Reply:
Handle properties are now supported.